connectors/src/connectors/slack/bot.ts

```diff
@@ -36,6 +36,14 @@ const { DUST_API = "https://dust.tt" } = process.env;
 
 class SlackExternalUserError extends Error {}
 
+const DEFAULT_ASSISTANTS_ORDER = [
```

Comment:
this sadly needs to be duplicated on front and connectors because we have no way to share it